src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symbol.java

```diff
@@ -1510,22 +1510,14 @@ public RecordComponent getRecordComponent(JCVariableDecl var, boolean addIfMissi
                  * case we need to use the position to disambiguate
```

Comment:
I'd suggest to consider renaming this method. It does not "get" the record component, it (re-)creates it The `addIfMissing` parameter appears to be always `true`, and can be removed.

Reply:
I agree